Userland/Utilities/cal.cpp

```diff
@@ -62,7 +63,12 @@ enum class Header {
     Month,
 };
 
-static ErrorOr<Vector<String>> month_lines_to_print(Header header_mode, int month, int year)
+enum class StartOfWeek {
+    Monday,
+    Sunday
+};
```

Comment:
well now I feel left out :P

Our GUI calendar supports setting any day as the start of the week, can we please have that instead? maybe even respect the system-wide configuration for that!

Reply:
That's an awesome idea! Done